src/berrysizeestimator.py

```diff
@@ -1,18 +1,260 @@
+from abc import ABC, abstractmethod
+import cv2 as cv
+from rembg import remove
+import numpy as np
+from imutils import perspective
+from imutils import contours
+from math import pi
+from math import sqrt
+import imutils
+from scipy.spatial import distance as dist
+import math
 
+class BerrySizeEstimatorAbstractClass(ABC):
+    """
+        Implementa i metodi astratti per misurare gli acini
 
+    """
 
+    def __init__(self, input_image):
+        self.input_image = input_image
 
-class BerrySizeEstimator():
-    """
-    Descrizione di cosa fa la classe...
-    """
+    @abstractmethod
```

Comment:
La classe mi sembra ben fatta, i metodi devono avere solo lettere minuscole. In python fay riferimento allo standard PEP8 per lo stile: [https://peps.python.org/pep-0008/](https://peps.python.org/pep-0008/)